game.py

```diff
@@ -20,7 +20,30 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if letter in snowman_word:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[letter] = True
+        else:
+            print(f"The letter '{letter}' is not in the word")
+            wrong_guesses_list.append(letter)
+
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+            print(f"Congratulations, you win!")
+            print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+            break
+
+        print_snowman_graphic(len(wrong_guesses_list))
+        print(f"Wrong guesses: {wrong_guesses_list}")
+
+    if len(wrong_guesses_list) == SNOWMAN_MAX_WRONG_GUESSES:
```

Comment:
The instinct here is great, but technically this conditional is very similar to the condition that ends the while loop above! And this is another reason the `return` statement would be more powerful than `break`! You would have handled the win condition within the loop and stopped the function there! This means we know that any code after the while loop indicates a loss in the game, so the extra conditional actually isn't necessary! All you would need is to print out the lose string.

In the example I gave above, the equivalent would be returning either `False` or `-1` if the value we are looking for isn't found. We don't need an extra conditional because if the program has made it all the way through the loop without returning, it knows it didn't find what it is looking for!